config/package_layout.json

```diff
@@ -10,11 +10,11 @@
         "checker_md_docs": "www/docs/checker_md_docs",
         "web_client": "www/scripts/codechecker-api",
         "web_client_plugins": "www/scripts/plugins",
-        "web_client_dojo": "www/scripts/plugins/dojo",
-        "web_client_marked": "www/scripts/plugins/marked",
-        "web_client_jsplumb": "www/scripts/plugins/jsplumb",
         "web_client_codemirror": "www/scripts/plugins/codemirror",
```

Comment:
The only change is the order of the records with the prefix "web_client_" (except for "web_client_plugins"), they are now alphabetical order.